Add issue templates for talk proposal

jimcal opened the pull request:
In order to facilitate the talk proposal process, I added the issue
templates to help community members submit talk proposals. Given we have
2 hrs most meetup, a combination of 2 feature talks or 4 lightning talks
should serve our community well.

Thanks SeattleJS & @Frijol for issue template inspiration.

jimcal committed:
Add issue templates for talk proposal

In order to facilitate the talk proposal process, I added the issue
templates to help community members submit talk proposals. Given we have
2 hrs most meetup, a combination of 2 feature talks or 4 lightning talks
should serve our community well.

terichadbourne commented:
I love that you included the Code of Conduct piece! 

Is there anything you want to call out for potential speakers to avoid? For example, as an attendee I'd be excited about educational talks and demos of cool stuff people have built but turned off by anything that came across as a product pitch.

Frijol commented on .github/ISSUE_TEMPLATE/proposal--featured-talk.md:
```diff
@@ -0,0 +1,20 @@
+- [Github]()
+- [Website]()
+
+* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
```
```suggestion
* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
* [ ] I'm new to giving talks and want to practice present to an organizer over a video call first
```

Frijol replied:
(just an idea)

jimcal replied:
I like it! I don't have the mentoring experience for new speakers (yet). Is this something we want to committing to support? I am excited about the idea, at the same time I want to have a healthy discussion even if the next three months no one request this.

Frijol replied:
it's a good question. I think I'd be able to hold that role for at least the first few/don't imagine it as a big timesuck but hard to really know

jimcal replied:
Thanks for willing to help! I will add this into the revision commit I am making.

Frijol commented on .github/ISSUE_TEMPLATE/proposal--lightning-talk.md:
```diff
@@ -0,0 +1,20 @@
+- [Github]()
+- [Website]()
+
+* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
```
```suggestion
* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
* [ ] I'm new to giving talks and want to practice present to an organizer over a video call first
```

jimcal replied:
Yes, I should put it in. And also just now I realized I paste the code of conduct from my own fork. Will fix this.

Frijol commented on .github/ISSUE_TEMPLATE/proposal--featured-talk.md:
```diff
@@ -0,0 +1,20 @@
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine.
+
+Include a short bio--anything that gives us an idea of why you're interested to speak on this subject. Feel free to include any relevant links.
```
```suggestion
Include a short bio--anything that gives us an idea of why you're interested to speak on this subject. Feel free to include any relevant links, especially to a video of you speaking if you have one.
```

Frijol commented on .github/ISSUE_TEMPLATE/proposal--lightning-talk.md:
```diff
@@ -0,0 +1,20 @@
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine.
+
+Include a short bio--anything that gives us an idea of why you're interested to speak on this subject. Feel free to include any relevant links.
```
```suggestion
Include a short bio--anything that gives us an idea of why you're interested to speak on this subject. Feel free to include any relevant links, especially to a video of you speaking if you have one.
```

Frijol reviewed (approved):
Love this. Do we need to credit SeattleJS somewhere for the templates?

jimcal commented:
> I love that you included the Code of Conduct piece!
> 
> Is there anything you want to call out for potential speakers to avoid? For example, as an attendee I'd be excited about educational talks and demos of cool stuff people have built but turned off by anything that came across as a product pitch.

Thanks @terichadbourne , that 's a really good point. How do we balance between encouraging talks and sharing ideas, and avoid the commercialized product pitch? 

I don't have a comprehensive view on this subject. Current ideas that comes to mind would be: 1. whether the code is open source; 2. whether the product/service has a commercial license available for purchase.

@Frijol @zebul feel free to chime in!

terichadbourne commented:
@jimcal I don't think that whether a talk feels pitchy is necessary related to whether it discusses a product that can be sold. For example, when I worked at Cloudant (an as-a-service version of the open source CouchDB), I gave talks/demos that showed people how to build offline-capable apps. I used Cloudant for the demo but brushed it off as "I chose to use Cloudant for this, but you could also use CouchDB directly if you know how to host your own blah blah..." I guess I'd say the distinction is in the tone and in having a goal of education as opposed to selling. ¯\_(ツ)_/¯

jimcal committed:
Update templates to incorporate review feedback

1. update code of conduct link.
2. adding reminder to avoid product pitches or vendor driven content.
3. adding reminder for potential speakers to provide past talk videos,
optional.
4. adding a checkbox to offer help to first time speakers, optional.

Frijol commented on .github/ISSUE_TEMPLATE/proposal--featured-talk.md:
```diff
@@ -0,0 +1,20 @@
+
+# Title of talk
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid the content like product pitches or vendor driven talk. 
```
```suggestion
A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid content like product pitches or vendor-driven talks. 
```

Frijol commented on .github/ISSUE_TEMPLATE/proposal--lightning-talk.md:
```diff
@@ -0,0 +1,21 @@
+
+# Title of talk
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid the content like product pitches or vendor driven talk. 
```
```suggestion
A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid content like product pitches or vendor-driven talks. 
```

jimcal commented:
Shout out to @bniedermeyer & SeattleJS for inspiration!
